website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "IDE glossary"
+id: ide-glossary
+description: "Develop, test, run, and build in the Cloud IDE. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly"
+sidebar_label: IDE glossary
+tags: [IDE]
+---
+
+The [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) is an interface tool for developers to effortlessly build, test, run, and version-control their dbt projects - all from the convenience of your browser. Use the Cloud IDE to compile dbt code into SQL and run it against your database directly -- no command line required!
+
+## Landing page
+
+The IDE is designed to streamline your workflow, and features a popular user interface and layout with all files and folders on the left and the editor on the right. 
+
+The basic layout of the IDE landing page is divided into the following areas:
+
+A &mdash; [Git repository link](#git-repository-link)<br />
+B &mdash; [Documentation site button](#documentation-site-button)<br />
+C &mdash; [Version Control Menu](#version-control-menu)<br />
+D &mdash; [File Explorer](#file-explorer)<br />
+E &mdash; [Command bar](#command-bar)<br />
+F &mdash; [IDE Status button](#ide-status-button)<br />
+G &mdash; [Create New File button](#create-new-file-button)<br />
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-landing.jpg" width="100%" title="The Cloud IDE landing page"/>
+
+
+#### Git Repository link 
+The Git Repository Link displays the name of the active branch. Clicking this element will hyperlink to your repository, pinned to that same branch.
+
+#### Documentation site button
+The Documentation Site button links to the static dbt Documentation site powered by the most recently generated dbt artifacts in the IDE. This button is clickable once a `dbt docs generate` command is successfully invoked in the Command Bar.  
+
+#### Version Control menu
+The Version Control menu contains all git-related elements in the IDE. The Git Actions Button will dynamically update with the relevant action for the state of your editor, including prompt to pull remote changes, commit and sync when reverted commit changes are in the editor, or create a merge/pull request when appropriate. The dropdown menu on the Git Actions Button allows users to revert all changes, refresh git state, create merge/pull requests, and change branches. Refer to [Version control basics](/docs/collaborate/git/version-control-basics#the-git-button-in-the-cloud-ide) for more info. 
+
+Below the Git Actions Button, all changes to files since the last commit will be listed in the **Changes** section. Clicking on any change will open the Git Diff View of the selected file, allowing the user to see the inline changes.
+
+#### File Explorer 
```

Comment:
Some of the actions listed below require you are not in _read-only_, which primarily only appears when you're on the `main` branch, or custom configured `main` branch, I wonder if some clarification or link to this `main` behavior in this section could clarify why some options aren't appearing for users who are confused